modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/MiniCart.js

```diff
@@ -119,14 +121,29 @@ function MiniCart({
 				});
 
 				onAddToCart(latestActionURLs, latestCartState);
-			}
-			catch (error) {
+			} catch (error) {
```

Comment:
Did you run the source formatter in this module? I think SF won't like it

Reply:
✅